docs/source/user-guide/cli.md

```diff
@@ -467,6 +474,66 @@ Available commands inside DataFusion CLI are:
+| "WatchID"      | 2            | 344064             | 4611692774829951781 | 9223363791697310021 | 3031680               |
++----------------+--------------+--------------------+---------------------+---------------------+-----------------------+
+3 rows in set. Query took 0.053 seconds.
+```
+
+The returned table has the following columns for each row for each column chunk
+in the file. Please refer to the [Parquet Documentation] for more information.
+
+[parquet documentation]: https://parquet.apache.org/
+
+| column_name             | data_type | Description                                                                                         |
+| ----------------------- | --------- | --------------------------------------------------------------------------------------------------- |
+| filename                | Utf8      | Name of the file                                                                                    |
+| row_group_id            | Int64     | Row group index the column chunk belongs to                                                         |
+| row_group_num_rows      | Int64     | Count of rows stored in the row group                                                               |
+| row_group_num_columns   | Int64     | Total number of columns in the row group (same for all row groups)                                  |
+| row_group_bytes         | Int64     | Number of bytes used to store the row group (not including metadata)                                |
+| column_id               | Int64     | ID of the column                                                                                    |
+| file_offset             | Int64     | Offset within the file that this column chunk's data begins                                         |
+| num_values              | Int64     | Total number of values in this column chunk                                                         |
+| path_in_schema          | Utf8      | "Path" (column name) of the column chunk in the schema                                              |
+| type                    | Utf8      | Parquet data type of the column chunk                                                               |
+| stats_min               | Utf8      | The minimum value for this column chunk, if stored in the statistics, cast to a string              |
+| stats_max               | Utf8      | The minimum value for this column chunk, if stored in the statistics, cast to a string              |
+| stats_null_count        | Int64     | Number of null values in this column chunk, if stored in the statistics                             |
+| stats_distinct_count    | Int64     | Number of distinct values in this column chunk, if stored in the statistics                         |
+| stats_min_value         | Utf8      | Same as `stats_min`                                                                                 |
```

Comment:
wondering if this duplicated fields needed?

Reply:
I don't honestly know why the seemingly duplicated columns are present. It was done initially to mirror duckdb which has them. Maybe we should investigate the reason why 🤔 

```sql
D create table foo as select * from parquet_metadata('./benchmarks/data/hits.parquet');
D describe table foo;
┌─────────────────────────┬─────────────┬─────────┬─────────┬─────────┬─────────┐
│       column_name       │ column_type │  null   │   key   │ default │  extra  │
│         varchar         │   varchar   │ varchar │ varchar │ varchar │ varchar │
├─────────────────────────┼─────────────┼─────────┼─────────┼─────────┼─────────┤
│ file_name               │ VARCHAR     │ YES     │         │         │         │
│ row_group_id            │ BIGINT      │ YES     │         │         │         │
│ row_group_num_rows      │ BIGINT      │ YES     │         │         │         │
│ row_group_num_columns   │ BIGINT      │ YES     │         │         │         │
│ row_group_bytes         │ BIGINT      │ YES     │         │         │         │
│ column_id               │ BIGINT      │ YES     │         │         │         │
│ file_offset             │ BIGINT      │ YES     │         │         │         │
│ num_values              │ BIGINT      │ YES     │         │         │         │
│ path_in_schema          │ VARCHAR     │ YES     │         │         │         │
│ type                    │ VARCHAR     │ YES     │         │         │         │
│ stats_min               │ VARCHAR     │ YES     │         │         │         │
│ stats_max               │ VARCHAR     │ YES     │         │         │         │
│ stats_null_count        │ BIGINT      │ YES     │         │         │         │
│ stats_distinct_count    │ BIGINT      │ YES     │         │         │         │
│ stats_min_value         │ VARCHAR     │ YES     │         │         │         │
│ stats_max_value         │ VARCHAR     │ YES     │         │         │         │
│ compression             │ VARCHAR     │ YES     │         │         │         │
│ encodings               │ VARCHAR     │ YES     │         │         │         │
│ index_page_offset       │ BIGINT      │ YES     │         │         │         │
│ dictionary_page_offset  │ BIGINT      │ YES     │         │         │         │
│ data_page_offset        │ BIGINT      │ YES     │         │         │         │
│ total_compressed_size   │ BIGINT      │ YES     │         │         │         │
│ total_uncompressed_size │ BIGINT      │ YES     │         │         │         │
├─────────────────────────┴─────────────┴─────────┴─────────┴─────────┴─────────┤
│ 23 rows                                                             6 columns │
└───────────────────────────────────────────────────────────────────────────────┘
```